__tests__/components/App.test.js

```diff
@@ -56,17 +56,20 @@ describe("#routes", () => {
     it.skip('/editor renders Editor component', () => {
       // Stub `Config.spoofSinopiaServer` static getter to force RT to come from server
       jest.spyOn(Config, 'spoofSinopiaServer', 'get').mockReturnValue(false)
-      saveState({loginJwt: Config.awsCognitoJWTHashForTest, isAuthenticated: true}, 'jwtAuth')
+      // TODO: This method is undefined
+      // saveState({loginJwt: Config.awsCognitoJWTHashForTest, isAuthenticated: true}, 'jwtAuth')
       const component = renderEditorRoute("/editor")
       expect(component.find(ImportResourceTemplate).length).toEqual(1)
     })
 
     // TODO: needs fixing for re-done authN approach, see #528
     it.skip('/templates renders the Login component (with location props) if the user is not authenticated', () => {
-      saveState({loginJwt: {}, isAuthenticated: false}, 'jwtAuth')
-      const component = renderRoutes("/templates")
-      expect(component.find(Login).length).toEqual(1)
-      expect(component.find(Login).prop('location')).toBeDefined()
+      // TODO: This method is undefined
+      // saveState({loginJwt: {}, isAuthenticated: false}, 'jwtAuth')
+      // const component = renderRoutes("/templates")
+      // TODO: Login is not defined or imported
```

Comment:
yeah, that component got removed in #460 (and the test needed reworking anyway, so skipped and punted w/ the intention of revisiting in #539).

Reply:
ok, np! Commented it out to make eslint happy. Resolving.